zebra-cli/Cargo.toml

```
@@ -0,0 +1,31 @@
+[package]
+name = "zebra-cli"
+authors = ["Zcash Foundation <zebra@zfnd.org>"]
+license = "MIT OR Apache-2.0"
+version = "1.0.0-alpha.0"
+edition = "2018"
+repository = "https://github.com/ZcashFoundation/zebra"
+
+[dependencies]
+gumdrop = "0.7"
+qrcode = "0.12.0"
+rand = "0.8.4"
+serde = { version = "1", features = ["serde_derive"] }
+thiserror = "1"
+
+zcash_address = { git = "https://github.com/ZcashFoundation/librustzcash.git", rev = "53d0a51d33a421cb76d3e3124d1e4c1c9036068e" }
+
+zebra-chain = { path = "../zebra-chain" }
+# zebra-client = { path = "../zebra-client" }
+
+[dependencies.abscissa_core]
+version = "0.5.2"
+# optional: use `gimli` to capture backtraces
+# see https://github.com/rust-lang/backtrace-rs/issues/189
+# features = ["gimli-backtrace"]
```

Nitpick: Consistency with other version specifiers.

```suggestion
abscissa_core = "0.5.2"
```